staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/cbor_test.go

```diff
@@ -156,6 +156,33 @@ func TestEncode(t *testing.T) {
 				}
 			},
 		},
+		{
```

Comment:
This was possible even before this PR, correct? So it's just additional data related to the new code IIUC

Reply:
Yes, although the behavior of the "list element missing kind", "list element missing version", and item GVK heuristic cases would have had different expectations since the original implementation had been calling SetUnstructuredContent.